weather/server.py

```diff
@@ -426,6 +427,9 @@ def api(port):
             print(' CANCEL \n')
     sys.stderr.write('\r ! apiServer Exit\n')
 
+def sigTermHandler(signum, frame):
+    sys.exit()
```

Comment:
My recommendation/preference would be to use `raise SystemExit` per explanation [here](https://github.com/jasonacox/Powerwall-Dashboard/issues/353?notification_referrer_id=NT_kwDOBnsFf7Q3NzcxNzQzMTA2OjEwODcyNTYzMQ#issuecomment-1728709935)